cluster/addons/podsecuritypolicies-rbac/gce/system-serviceaccount-binding.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: rbac.authorization.k8s.io/v1alpha1
+kind: ClusterRoleBinding
+metadata:
+  name: system:service-accounts-podsecuritypolicy:restricted
+  labels:
+    kubernetes.io/cluster-service: "true"
+    addonmanager.kubernetes.io/mode: Reconcile
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: podsecuritypolicy:restricted
+subjects:
+- apiVersion: rbac.authorization.k8s.io/v1alpha1
+  kind: Group
+  name: system:serviceaccounts
```

Comment:
You probably want normal users to have this as well. Admission can prevent a user from execing into a pod they would not have been allowed to create via PSP